documentation/modules/rn-2.4.adoc

```diff
@@ -138,11 +145,21 @@ Improve the automatic renaming of VMs during migration to fit RFC 1123. This fea
 
 .Prevent locking user accounts due to incorrect credentials
 
-If a user specifies an incorrect password for RHV providers, they are no longer locked in RHV. An error returns when the RHV manager is accessible and adding the provider. If the RHV manager is not accessible, the provider is added, but there would be no further attempt after failing, due to incorrect credentials. link:https://issues.redhat.com/browse/MTV-324[(MTV-324)]
+If a user specifies an incorrect password for {rhv-short} providers, they are no longer locked in {rhv-short}. An error returns when the {rhv-short} manager is accessible and adding the provider. If the {rhv-short} manager is inaccessible, the provider is added, but there would be no further attempt after failing, due to incorrect credentials. link:https://issues.redhat.com/browse/MTV-324[(MTV-324)]
 
 .Users without cluster-admin role can create new providers
-Previously, the `cluster-admin` role was required to browse and create providers. In this release, users with sufficient permissions on MTV resources (providers, plans, migrations, networkmaps, storagemaps, hooks) can operate MTV without cluster-admin permissions. link:https://issues.redhat.com/browse/MTV-334[(MTV-334)]
+
+Previously, the `cluster-admin` role was required to browse and create providers. In this release, users with sufficient permissions on MTV resources (providers, plans, migrations, NetworkMaps, storage maps, hooks) can operate MTV without cluster-admin permissions. link:https://issues.redhat.com/browse/MTV-334[(MTV-334)]
 
 .Convert i440fx to q35
 
 Migration of virtual machines with i440fx chipset is now supported. The chipset is converted to q35 during the migration. link:https://issues.redhat.com/browse/MTV-430[(MTV-430)]
+
+.Preserve the UUID setting in SMBIOS for a VM that is migrated from {rhv-short}
+
+The Universal Unique ID (UUID) number within the System Management BIOS (SMBIOS) no longer changes for VMs that are migrated from {rhv-short}. This enhancement enables applications that operate within the guest operating system and rely on this setting, such as for licensing purposes, to operate on the target OCP cluster in a manner similar to that of {rhv-short}. link:https://issues.redhat.com/browse/MTV-597[(MTV-597)]
+
+.Do not expose password for {rhv-short} in error messages
+
+Previously, the password that was specified for ovirt-engine appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for ovirt-engine.
```

Comment:
@RichardHoch suggested to use the downstream term for RHV here instead of ovirt-engine, which makes sense. for consistency we can use RHV manager as we do elsewhere in the release notes
```suggestion
Previously, the password that was specified for {rhv-short} manager appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for {rhv-short} manager.
```